x-pack/test/api_integration/apis/uptime/rest/snapshot.ts

```diff
@@ -6,47 +6,97 @@
 
 import { expectFixtureEql } from '../graphql/helpers/expect_fixture_eql';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { makeChecksWithStatus, getChecksDateRange } from '../graphql/helpers/make_checks';
 
 export default function({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
 
   describe('snapshot count', () => {
-    let dateRangeStart = '2019-01-28T17:40:08.078Z';
-    let dateRangeEnd = '2025-01-28T19:00:16.078Z';
-
-    it('will fetch the full set of snapshot counts', async () => {
-      const apiResponse = await supertest.get(
-        `/api/uptime/snapshot/count?dateRangeStart=${dateRangeStart}&dateRangeEnd=${dateRangeEnd}`
-      );
-      expectFixtureEql(apiResponse.body, 'snapshot');
-    });
+    const dateRangeStart = new Date().toISOString();
+    const dateRangeEnd = new Date().toISOString();
 
-    it('will fetch a monitor snapshot filtered by down status', async () => {
-      const filters = `{"bool":{"must":[{"match":{"monitor.status":{"query":"down","operator":"and"}}}]}}`;
-      const statusFilter = 'down';
-      const apiResponse = await supertest.get(
-        `/api/uptime/snapshot/count?dateRangeStart=${dateRangeStart}&dateRangeEnd=${dateRangeEnd}&filters=${filters}&statusFilter=${statusFilter}`
-      );
-      expectFixtureEql(apiResponse.body, 'snapshot_filtered_by_down');
-    });
+    describe('when no data is present', async () => {
+      it('returns a null snapshot', async () => {
+        const apiResponse = await supertest.get(
+          `/api/uptime/snapshot/count?dateRangeStart=${dateRangeStart}&dateRangeEnd=${dateRangeEnd}&filters=`
```

Comment:
Why add the `filters` param? It's a `maybe` param, so we can omit it if we have no filter.